src/packaging/bin/cassandra-reaper

```diff
@@ -23,7 +23,7 @@ fi
 
 if [ -z "$CLASS_PATH" ]; then
   echo "Looking for reaper in /usr/share/cassandra-reaper/"
```

Comment:
Btw I just noticed that the path that is echoed here doesn't match where it is looking (line just below) :wink: